tests/bgp/test_bgp_speaker.py

```diff
@@ -164,6 +164,9 @@ def common_setup_teardown(duthost, ptfhost, localhost):
 
     for ip in vlan_ips:
         duthost.command("ip route flush %s/32" % ip.ip, module_ignore_errors=True)
+        duthost.command("arp -d %s" % ip.ip, module_ignore_errors=True)
```

Comment:
I think we can just flush arp since this line 116 is anyways doing a flush before the test and all entries are added by the test. What do you think?

Reply:
Done